lib/cretonne/src/ir/dfg.rs

```diff
@@ -106,6 +106,23 @@ impl DataFlowGraph {
     }
 }
 
+/// Resolve value aliases.
+///
+/// Find the original SSA value that `value` aliases.
+fn resolve_aliases(values: &EntityMap<Value, ValueData>, value: Value) -> Value {
```

Comment:
Nice. I figured something like this would be necessary to make the borrow checker happy.